test/powershell/Modules/Microsoft.PowerShell.Utility/Json.Tests.ps1

```diff
@@ -1306,7 +1313,7 @@ Describe "Validate Json serialization" -Tags "CI" {
             ValidateProperties -serialized $result.SerializedViaJson -expected $result.Expected -properties $propertiesToValidate
         }
 
-        It "Validate 'Get-Command Get-help' output with Json conversion" {
+        It "Validate 'Get-Command Get-help' output with Json conversion" -Pending:(!$notNewConvertToJson) {
```

Comment:
Could you use `Set-ItResult` with a `-Because` so we know why this is pending. Similar for other pending tests.

Reply:
All pendings are the consequences of breaking changes.

For each we should make a conclusion - fix or ignore. I need this feedback from reviewers and PowerShell Committee.
If fix - in the PR or follow?